build.gradle

```diff
@@ -15,7 +15,7 @@ java {
     }
 }
 
-ext.pitestAggregatorVersion = "1.15.0"   //Must be equal to default PIT version in PitestPlugin
+ext.pitestAggregatorVersion = "1.20.0"   //Must be equal to default PIT version in PitestPlugin
```

Comment:
It's interesting that all tests pass in your fork. In upstream, it would [fail the build](https://github.com/szpak/gradle-pitest-plugin/blob/a7668afb33ec77f9ae60528672e644ae2cb50868/build.gradle#L6) to prevent incompatibility between "default PIT version" and this one :thinking: